test/Libraries/RevitTestServices/RevitNodeTestBase.cs

```diff
@@ -43,8 +43,7 @@ public override void Setup()
 
         protected override TestSessionConfiguration GetTestSessionConfiguration()
         {
-            var asmDir = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
-            return new TestSessionConfiguration(Path.GetFullPath(asmDir + @"\..\"), asmDir);
+            return new TestSessionConfiguration(Dynamo.Applications.DynamoRevitApp.DynamoCorePath);
```

Comment:
OK. So I finally come back to this. From my testing, using version selector will result in some error when actually running the test case using RTF that `Could not load file or assembly SystemTestServices`. So specifying the particular DynamoRevitDS.dll will be the recommended way